### ISSUE TITLE
Remove - of jvm_route module. Why ?

### ISSUE THREAD

**Issue by radut** [yaoweibin/nginx_ajp_module#19]
Hi,
Might you be so kind as to tell me why you removed jvm_route module ?
That was the #1 benefit of the nginx_ajp_module..

**Comment by baux1980** [comment]
You are right! I've modified nginx ajp module to work with jvm_route and newer nginx versions, come and see:

https://github.com/baux1980/nginx_ajp_module
